Author: Fathi Boudra <fabo@debian.org>

Upstream clear default CFLAGS and force -O3.
Use default CFLAGS on Debian.

--- a/configure.in
+++ b/configure.in
@@ -154,11 +154,11 @@ AC_CHECK_HEADERS(linux/compiler.h linux/
 
 
 dnl Clear default CFLAGS
-if test x"$CFLAGS" = x"-g -O2"; then
-  CFLAGS=
-fi
+#if test x"$CFLAGS" = x"-g -O2"; then
+#  CFLAGS=
+#fi
 
-CFLAGS="-O3 -ffast-math -pipe $CFLAGS"
+CFLAGS="$CFLAGS -ffast-math -pipe"
 
 DFB_INTERNAL_CFLAGS="-D_GNU_SOURCE $DFB_INTERNAL_CFLAGS"
 
